src/pages/Home/components/Days/Card.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Day } from './Days'
+import {GlobalSvgSelector} from "../../../../assets/icons/global/GlobalSvgSelector";
+import {CardStyle, DayInfo, DayStyle, Img, DayTemp, NightTemp, Info} from "./DaysStyles";
+interface Props{
```

Comment:
Remove interfaces to ~types/interfaces.ts 
```suggestion
interface CardComponentProps {
```